### DISCUSSION TITLE
安装telescope的时候，执行checkhealth telescope报错了

### DISCUSSION THREAD

**Discussion by chernolll** [nshen/learn-neovim-lua#157] [Q&A]
报错信息如下：
```txt
Error: No healthcheck found for 'telescope' plugin
```

不管是PackerSync安装的，还是自己手动git clone 下来，都是这样

插件都是下载成功的
<img width="788" alt="image" src="https://user-images.githubusercontent.com/46297020/213114029-f54a1d43-3c7d-4e79-add8-65994dd1632f.png">

**Comment by chernolll** [comment]
<img width="980" alt="image" src="https://user-images.githubusercontent.com/46297020/213115832-443e1a32-cc22-4ff6-8502-e8156772e58c.png">

执行`Telescope find_files`的时候也会报错；重装很多次了

**Reply by nshen** [reply]
neovim 版本问题？ 我的0.8.x是正常的。

这样安装的吗？
```
use {
  'nvim-telescope/telescope.nvim', tag = '0.1.1',
  requires = { {'nvim-lua/plenary.nvim'} }
}
```

**Reply by chernolll** [reply]
还是一样的报错，作者方便加个联系方式吗，github不太方便询问（WeChat：W1213586653）

**Reply by nshen** [reply]
关键我也没遇到过这个错误，不知道怎么解决。你只能到telescope.nvim GitHub上英文发issues了 ，😭

**Reply by nshen** [reply]
昨天看到群里好像有人说是依赖 plenary.nvim 版本不对

**Reply by chernolll** [reply]
我那些报错的插件，全都回退版本，已经能用了，^_^
